depends/config.site.in

```diff
@@ -62,7 +62,7 @@ if test -z "$enable_zmq" && test -n "@no_zmq@"; then
   enable_zmq=no
 fi
 
-if test "x@host_os@" = xdarwin; then
+if test "@host_os@" = darwin; then
```

Comment:
How is this related to this change

Reply:
New [SC2268](https://github.com/koalaman/shellcheck/wiki/SC2268) added.